src/random.cpp

```diff
@@ -255,6 +295,11 @@ void GetStrongRandBytes(unsigned char* out, int num)
     GetOSRand(buf);
     hasher.Write(buf, 32);
 
+    // Third source: HW RNG, if available.
+    if (GetHWRand(buf)) {
+        hasher.Write(buf, 32);
```

Comment:
I guess logging (`Using RdRand as entropy source`) at this point would make more sense.